requirements-dev-frozen.txt

```diff
@@ -187,6 +187,7 @@ tqdm==4.64.1
 trio==0.21.0
 trio-websocket==0.9.2
 types-beautifulsoup4==4.11.6
+types-cachetools==5.3.0.5
```

Comment:
Did we have nothing already that could do something like this?

Reply:
in fact we did (it's this library!) -- this is just the type stubs for it

Reply:
oh doy! haha sorry!